modules/doc/content/training/more_detail/PHYSOR_MS_2024_04_21.md

```diff
@@ -0,0 +1,32 @@
+#### ART/NEAMS M&S Molten Salt Reactors @PHYSOR24 April 21, 2024
+
+!style! halign=left
+!include training/more_detail/registration_link.md
+!style-end!
+
+#### Course Description
+
+!style! halign=left
+The goal of the training session is to gain familiarity modeling the running-in phase of pebble bed
+reactors (PBRs) with the MOOSE-based tools Griffin and Pronghorn. Griffin includes capabilities for
```

Comment:
I think this may be using the wrong description? Both mention PBRs, while the title of this page refers to MSRs.

Reply:
Tell me about it... these descriptions are direct copies from: https://ncrcaims.inl.gov/Identity/Account/TrainingRegistration

I'll make the change regardless. 🍻

Reply:
Actually you are right, good catch! I copied the same text by mistake or something...